channels/facebook/wphook.js

```diff
@@ -19,6 +19,10 @@ const rpn = require('request-promise-native')
 const WORKPLACE_VERIFY_TOKEN = process.env.WORKPLACE_VERIFY_TOKEN;
 const WORKPLACE_PAGE_ACCESS_TOKEN = process.env.WORKPLACE_PAGE_ACCESS_TOKEN
 
+var startChannel= () => {
```

Comment:
is this function neccessary?

Reply:
No and yes:
For WP and FBM we don't need to. However, I'm trying to conceptualize common methods across all channels. startChannel is used today with Nexmo, and I suspect that in the future we would to som something with WP/FBM as well